pkg/reconciler/route/controller.go

```diff
@@ -60,6 +61,7 @@ func newController(
 ) *controller.Impl {
 	logger := logging.FromContext(ctx)
 	serviceInformer := serviceinformer.Get(ctx)
+	endpointsInformer := endpointsinformer.Get(ctx)
```

Comment:
We don't add event handler `endpointsInformer.Informer().AddEventHandler(handleControllerOf)` for the endpoints?

Reply:
I thought about but thought it wasn't necessary since

1) Users will probably not have access to modify this endpoint object (because of https://github.com/kubernetes/kubernetes/issues/103675) - and we have the option for the visibility label on the service
2) There's no information from the endpoint being propagated to the route


Though we may want to 'scope' the endpoint informer to only watch endpoints with that controller route label